wallet-new/src/Cardano/Wallet/API/V1/Addresses.hs

```diff
@@ -13,3 +14,6 @@ type API = "addresses" :> WalletRequestParams
       :<|> "addresses" :> ReqBody '[JSON] Address
                        :> Summary "Creates a new Address."
                        :> Post '[JSON] Address
+      :<|> "addresses" :> Capture "address" Text
+                       :> Summary "Checks the validity of an address."
+                       :> Get '[JSON] AddressValidity
```

Comment:
I'm wondering if this is better expresses as a `PUT`, because technically this is idempotent and doesn't query directly the model. On the other hand, it feels artificial, because it's really not an update of the model either. @DominikGuzei any preference from you folks?

Reply:
Hmm I think one of my commits got lost somewhere because this is actually wrong. Should be a capture and then "validity" after the capture (GET /addresses/x/validity). I am not sure about the PUT in this context though I usually associate PUTs with having some effect on the state like persisting some value.